### DISCUSSION TITLE
Regarding Static IP for Github Renovate app

### DISCUSSION THREAD

**Discussion by tgill2** [renovatebot/renovate#27479] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

We're planning to enforce IP restrictions on our Github organizations. Is there any static IP range for the Renovate app that we can whitelist ? We want to ensure it can still function smoothly after the restrictions are applied.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment]
There is a static IP for the app, but I think the way app restrictions work it needs us to configure it as part of the app's own settings. We have been waiting for a weekend when we can test the setting and confirm (hopefully) that it doesn't cause an outage.

**Reply by tgill2** [reply]
Thanks for confirming , just wanted to check if there is any estimated timeline when we can expect the Renovate app to be configured with the static IP.

**Reply by rarkins** [reply]
It's been "maybe next week" for a while. We'll see if we can test this in the next couple of weeks
